templates/templates/navigation-enterprise-h.html

```diff
@@ -55,7 +55,7 @@ <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
             <li class="p-list__item"><a href="/kubernetes/features">Multicloud K8s on bare metal, VMware, and all public clouds</a></li>
             <li class="p-list__item"><a href="/kubernetes/managed">Fully managed K8s</a></li>
             <li class="p-list__item"><a href="https://microk8s.io" title="Visit the MicroK8s website - external site" >Get K8s for your workstation and edge / IoT</a></li>
-            <li class="p-list__item"><a href="/ai/what-is-kubeflow">AI and ML on Kubernetes</a></li>
+            <li class="p-list__item"><a href="/ai/">AI and ML on Kubernetes</a></li>
```

Comment:
could you remove the trailing slash to keep in line with the other links?
sorry for being pedantic! ;)